src/main/java/ru/yandex/practicum/filmorate/controller/FilmController.java

```diff
@@ -1,47 +1,65 @@
 package ru.yandex.practicum.filmorate.controller;
 
-import lombok.extern.slf4j.Slf4j;
 import org.springframework.web.bind.annotation.*;
 import ru.yandex.practicum.filmorate.model.Film;
+import ru.yandex.practicum.filmorate.service.FilmService;
 
 import javax.validation.Valid;
-import javax.validation.ValidationException;
 import java.util.Collection;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.Optional;
 
 @RestController
 @RequestMapping("/films")
-@Slf4j
 public class FilmController {
 
-    private final Map<Integer, Film> films = new HashMap<>();
+    private final FilmService filmService;
 
-    private int id = 1;
+    public FilmController(FilmService filmService) {
+        this.filmService = filmService;
+    }
+
+    @GetMapping("/{id}")
+    @ResponseBody
+    public Film getById(@PathVariable long id) {
+        return filmService.getById(id);
+    }
 
     @GetMapping
+    @ResponseBody
```

Comment:
ResponseBody в классах-контроллерах не нужен.  Тут объяснение:
https://www.baeldung.com/spring-request-response-body
`Remember, we don't need to annotate the @RestController-annotated controllers with the @ResponseBody annotation since Spring does it by default.`